NOTICE

```diff
@@ -0,0 +1,217 @@
+Grafana Beyla 
+Copyright Grafana Labs
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention by opening a GitHub issue at 
+https://github.com/grafana/beyla/issues.
```

Comment:
You are missing notices for `libbpf` and a lot of Go modules.

Reply:
OK, sounds good. I'll add them. Thanks for checking.

Reply:
We have a third_party_licenses.csv for all of the third-party libraries we use, which is generated with the Makefile. I added the file as well to the binaries and two additional notices for libbpf (vmlinux.h) and parts of ebpf-go we have in our include.

Reply:
`third_party_licenses.csv` includes license notices, but it misses copywrite notices.

Reply:
I added wording to say that the copyright for the third party libraries belongs to the authors of the third party libraries.

Reply:
Probably it would be the best if you check this with legals. Apart from it LGTM.